src/menus/SelectMenus.cpp

```diff
@@ -166,26 +166,19 @@ void SeekWhenAudioActive(double seekStep, wxLongLong &lastSelectionAdjustment)
 double GridMove
 (AudacityProject &project, double t, int minPix)
 {
-   auto &formats = ProjectNumericFormats::Get(project);
```

Comment:
Good to see this; now, you can remove "ProjectNumericFormats.h" and "NumericConvert.h" from inclusions.

Reply:
done